docs/articles/documentation/using-testcafe/common-concepts/browser-support.md

```diff
@@ -88,6 +92,83 @@ You can search npm for plugins developed by the community. Their names begin wit
 
 You can also create your own plugin. See [Browser Provider Plugin](../../extending-testcafe/browser-provider-plugin/) for instructions.
 
+## Using Chrome-specific Features
+
+Using the [Chrome Debugging Protocol (CDP)](https://chromedevtools.github.io/devtools-protocol/), TestCafe provides the following Chrome-specific features:
+
+* [Running Tests in Headless Mode](#running-tests-in-headless-mode)
+* [Running Tests in the Device Emulation Mode](#running-tests-in-the-device-emulation-mode)
+
+CDP requires a remote debugging port to attach to a Chrome instance. TestCafe automatically assigns a free port but you can specify a custom port via the `cdpPort` argument if necessary:
+
+```sh
+testcafe "chrome:headless:cdpPort=9223" tests/sample-fixture.js
+```
+
+### Running Tests in Headless Mode
+
+TestCafe allows you to run tests in Google Chrome without any visible UI shell - in the [headless mode](https://developers.google.com/web/updates/2017/04/headless-chrome). Note that the [Take screenshot](../../test-api/actions/take-screenshot.html) and [Resize window](../../test-api/actions/resize-window.html) features are fully supported in headless mode.
+
+```sh
+testcafe "chrome:headless" tests/sample-fixture.js
+```
+
+> Note that headless mode is available for Mac and Linux in Chrome 59. You need a [Canary build](https://www.google.com/chrome/browser/canary.html) to enable headless mode on Windows.
+
+Specify a path to installation location, if you install a portable version of Chrome:
+
+```sh
+testcafe "chrome:path/to/chrome:headless" tests/sample-fixture.js
+```
+
```

Comment:
In headless mode the browser's viewport size is 1280x800 by default. You can change it via the (ResizeWindow)[../../test-api/actions/resize-window.html] command or via [Device Emulation Mode]() parameters.